go/vt/workflow/sleep_workflow.go

```diff
@@ -69,7 +69,8 @@ func (sw *SleepWorkflow) Run(ctx context.Context, manager *Manager, wi *topo.Wor
 	sw.mu.Lock()
 	sw.manager = manager
 	sw.wi = wi
-	sw.node = NewToplevelNode(wi, sw)
+	sw.node = NewNode()
+	sw.node.AttachToWorkflow(wi, sw)
```

Comment:
Maybe this could be done inside AddRootNode, and that would simplify the API even more? I can't see a reason why we would call this but not AddRootNode, or vice versa.